### ISSUE TITLE
Deployer callback URL - Laravel error

### ISSUE THREAD

**Issue by axeloz** [REBELinBLUE/deployer#225]
### Expected behaviour

When calling the callback URL in GET, Deployer should start a new deployment.

### Actual behaviour

The provided URL returns a Laravel error

### Steps to reproduce

Copy/paste URL in browser.

### Environment info
Operating System: Ubuntu
PHP Version: 5.5

### Logs (see storage/logs/) or other output that would be helpful
```
MethodNotAllowedHttpException in RouteCollection.php line 219:
in RouteCollection.php line 219
at RouteCollection->methodNotAllowed(array('POST')) in RouteCollection.php line 206
at RouteCollection->getRouteForMethods(object(Request), array('POST')) in RouteCollection.php line 158
at RouteCollection->match(object(Request)) in Router.php line 823
at Router->findRoute(object(Request)) in Router.php line 691
at Router->dispatchToRoute(object(Request)) in Router.php line 675
at Router->dispatch(object(Request)) in Kernel.php line 246
at Kernel->Illuminate\Foundation\Http\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 52
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in TrustProxies.php line 46
at TrustProxies->handle(object(Request), object(Closure))
at call_user_func_array(array(object(TrustProxies), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 136
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 32
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in CheckForMaintenanceMode.php line 44
at CheckForMaintenanceMode->handle(object(Request), object(Closure))
at call_user_func_array(array(object(CheckForMaintenanceMode), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 136
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 32
at Pipeline->Illuminate\Routing\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 103
at Pipeline->then(object(Closure)) in Kernel.php line 132
at Kernel->sendRequestThroughRouter(object(Request)) in Kernel.php line 99
at Kernel->handle(object(Request)) in index.php line 53
```

**Comment by axeloz** [comment]
My mistake, just checked in the routes, POST requests only
not a bug

**Comment by axeloz** [comment]
There seems to be kind of a problem anyways:
Now it's working but Deployer has just sent 1000 notifications on our Slack channel saying that the Heartbeat was down. Maybe there should be only one notification by status change or at most one notification per 6 hours. 
I have had to generate a new Slack callback URL to make it stop.

**Comment by REBELinBLUE** [comment]
Duplicate of #212

**Comment by REBELinBLUE** [comment]
Hang on, actually that isn't a duplicate of that thinking about it. Are you saying as soon as you enabled the cronjob it sent 1000 notifications?

**Comment by axeloz** [comment]
@REBELinBLUE yes, that is what I said. As if the queue was filled of notifications during days and days then flushed when Cron was launched.

**Comment by REBELinBLUE** [comment]
That is very weird, because if the crib job wasn't running it wouldn't have queued anything up and the queue is dispatched immediately so they shouldn't back up like that. I wonder if it is actually the cronjob firing for each time which is missed rather than just the last time

Stephen Ball
stephen@rebelinblue.com

**Comment by axeloz** [comment]
Hard to know because Slack posts aren't dated. I mean, they are dated when posted but don't display: "yesterday 10AM: heartbeat failed". 
(which could be an improvement idea BTW). 
How can I help you debug this?

**Comment by REBELinBLUE** [comment]
No it's OK, I can test if it is that myself

**Comment by REBELinBLUE** [comment]
Hmm, so I am unable to recreate this, so hopefully it'll be fixed with #212

**Comment by axeloz** [comment]
OK thanks, I'll try to reproduce the problem one of these days....
